sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -145,6 +191,45 @@ const ListingForm = ({ listing, editMode }: ListingFormProps) => {
     setAlert(null)
     setLoading(true)
     try {
+      // data cleanup and formatting
+      data.disableUnitsAccordion = stringToBoolean(data.disableUnitsAccordion)
+      units.forEach((unit) => {
+        switch (unit.unitType) {
+          case "threeBdrm":
+            unit.numBedrooms = 3
+            break
+          case "twoBdrm":
+            unit.numBedrooms = 2
+            break
+          default:
```

Comment:
In current listing jsons, in some cases where it's a studio we are setting numBedrooms to null and in other cases we are setting it to 1, not sure which case should be settled on, but studios kinda sorta don't have bedrooms but kinda do?

Reply:
Updated per your suggestion.